### DISCUSSION TITLE
Menu block Style selector

### DISCUSSION THREAD

**Discussion by X3Technology** [vanjarosoftware/Vanjaro.Platform#1694] [Q&A] [answered]
![menu_style](https://user-images.githubusercontent.com/1585717/131218754-b06f4bc7-c9c5-45f9-b807-d99cb70809d9.PNG)
 I need to be able to add my custom style to this dropdown so I can style it properly with CSS.  I want to create a vertical menu for a sidebar.

**Comment by meetmandeep** [comment, marked as answer]
Simply copy the Default.cshtml and rename it to Vertical.cshtml at \Website\Portals\_default\vThemes\Basic\blocks\Menu\Templates

You have completed control over the rendered HTML output so feel free to change Vertical.cshtml as needed. Be sure to restart application for cache to clear and Vertical to appear as a new style.

**Comment by X3Technology** [comment]
Excellent.  Thanks!
